src/components/MovieDetails/MovieDetails.jsx

```diff
@@ -34,12 +39,12 @@ export default MovieDetails;
 
 MovieDetails.propTypes = {
   movieInfo: PropTypes.shape({
-    imageUrl: PropTypes.string,
-    name: PropTypes.string,
-    releaseYear: PropTypes.number,
-    rating: PropTypes.number,
-    duration: PropTypes.string,
-    description: PropTypes.string,
+    poster_path: PropTypes.string,
```

Comment:
I understand your wish to change component props based on the BE data)))) But the real projects are not ideal, so BE data and FE interfaces are not equal. So, we need to transform BE data for FE needs on your side. You can create a function-transformer and use it after data getting.